core-features/tracing/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -eux
+
+yarn install
+yarn prisma generate
+
+docker-compose up -d
+
+node app.js
```

Comment:
Tests are currently failing with 
```
+ node app.js
Warning: Attempt 2 failed. Reason: Timeout of 600000ms hit
```

```suggestion
node app.js &
```
Adding `&` should not wait indefinitely = background process (see example like https://linuxhint.com/run-background-command-bash/)